tests/ToolInfoTest.php

```diff
@@ -21,13 +21,87 @@
  */
 final class ToolInfoTest extends \PHPUnit_Framework_TestCase
 {
+    const PHPUNIT_BIN = '/../vendor/bin/phpunit';
+
+    protected function setUp()
```

Comment:
I'm not sure using `setUp`/`tearDown` is relevant: the created symlinks are only used in `testIsInstalledByComposer`, maybe it would be easier to handle them in the test directly?